charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -52,3 +52,12 @@ spec:
                 fieldPath: metadata.namespace
           - name: TZ
             value: {{ .Values.timezone | default "UTC" }}
+        volumeMounts:
+        - mountPath: /var/lib/tls
+          name: tls
+          readOnly: true
+      volumes:
+      - name: tls
+        secret:
+          defaultMode: 420
+          secretName: client-tls
```

Comment:
Do we need to create the `client-tls ` secret manually?

Reply:
As of this PR, Yes, and the cert need to be signed by the k8s CA by `kubectl certificate approve`, as described in the PR OP.

PR #782 implements the automatic process of generating certs.

Reply:
make this optional, otherwise, it will fail when users don't need to enable TLS